array-methods.js

```diff
@@ -24,7 +30,15 @@ var hundredThousandairs = null;
     }
   assign the resulting new array to `datasetWithRoundedDollar`
 */
-var datasetWithRoundedDollar = null;
+const datasetWithRoundedDollar = bankBalances.map(currentObject => {
+
+  return currentObject = {
```

Comment:
With map, we should always be returning a "new" object, so setting the given currentObject and redefining it is not optimal here.